Plugin.pm

```diff
@@ -1469,6 +1477,22 @@ sub _trackItem {
 		$item->{year} = $track->{album}->{year} || substr($track->{$album}->{release_date_stream},0,4) || 0;
 	}
 
+	if ( $prefs->get('parentalWarning') && $track->{parental_warning} ) {
+		$item->{name} .= ' [E]';
+		$item->{line1} .= ' [E]';
+	}
+
+	if ( $prefs->get('discNumber') && $track->{album}->{media_count} > 1 && $prefs->get('trackNumber') ) {
+		$item->{name} = "Disc $track->{media_number}/$track->{track_number}: $item->{name}";
+		$item->{line1} = "Disc $track->{media_number}/$track->{track_number}: $item->{line1}";
+	} elsif ( $prefs->get('discNumber') && $track->{album}->{media_count} > 1 ) {
+		$item->{name} = "Disc $track->{media_number}: $item->{name}";
+		$item->{line1} = "Disc $track->{media_number}: $item->{line1}";
+	} elsif ( $prefs->get('trackNumber') ) {
+		$item->{name} = "$track->{track_number}: $item->{name}";
+		$item->{line1} = "$track->{track_number}: $item->{line1}";
+	}
```

Comment:
"Disc" should be localizable - or not used. Why not stick closer to what LMS does for the old UIs: `DISC-TRACKNUM. TITLE`? Or even re-use `Slim::Music::Info::addDiscNumberToAlbumTitle()`?